src/constants/blockchains.ts

```diff
@@ -61,6 +63,24 @@ const BLOCKCHAINS: {[chain in SupportedChains]: IBlockchainObject} = {
       raw: `https://rinkeby.etherscan.io/getRawTx?tx=${TRANSACTION_ID_PLACEHOLDER}`
     }
   },
+  [SupportedChains.Ethgoerli]: {
+    code: SupportedChains.Ethgoerli,
+    name: 'Ethereum Testnet',
+    signatureValue: 'ethereumGoerli',
+    transactionTemplates: {
+      full: `https://goerli.etherscan.io/tx/${TRANSACTION_ID_PLACEHOLDER}`,
+      raw: `https://goerli.etherscan.io/getRawTx?tx=${TRANSACTION_ID_PLACEHOLDER}`
+    }
+  },
+  [SupportedChains.Ethsepolia]: {
+    code: SupportedChains.Ethsepolia,
+    name: 'Ethereum Testnet',
+    signatureValue: 'ethereumSepolia',
+    transactionTemplates: {
+      full: `https://sepolia.etherscan.io/tx/${TRANSACTION_ID_PLACEHOLDER}`,
+      raw: `https://sepolia.etherscan.io/getRawTx?tx=${TRANSACTION_ID_PLACEHOLDER}`
+    }
+  },
```

Comment:
Mental note to get this data from explorer lookup

Reply:
Sorry, Your suggestion meant that there were duplicated same constants across `explorer-lookup` and this repo, so It would be nice to remove the duplicated by changing to get the constants from `explorer-lookup` like the below, right?

`src/constants/blockchains.ts`:
```src/constants/blockchains.ts
import { SupportedChains, BLOCKCHAINS, IBlockchainObject } from '@blockcerts/explorer-lookup';
// TODO: exports these variables on explorer-lookup/src/index.ts

export {
  SupportedChains,
  IBlockchainObject,
  BLOCKCHAINS,
};
```